examples/response/presentation_submission_multiple_vps.json

```diff
@@ -4,16 +4,16 @@
   "descriptor_map": [
     {
       "id": "ID Card with constraints",
-      "format": "ldp_vp",
+      "format": "di_vp",
       "path": "$[0]",
       "path_nested": {
-        "format": "ldp_vc",
+        "format": "di_vc",
         "path": "$.verifiableCredential[0]"
       }
     },
     {
       "id": "Example credential disclosing only address",
-      "format": "vc+sd-jwt",
+      "format": "sdjwt_vc",
```

Comment:
Can you explain this change please? It appears to be out of scope for this issue and PR.

Reply:
this one can be reverted it was a mistake